### DISCUSSION TITLE
Busco Equipo

### DISCUSSION THREAD

**Discussion by DavidSaboredo** [Launch-X-Latam/MisionFrontEnd#554] [Team Posts]
Hola gente que tal espero que esten bien, busco equipo para el hackaton 
_correo es  :lx06085@innovaccion.mx 

Saludos a todos.

**Comment by JosueDuenas1995** [comment]
Hola tanbien busco equipo.
Mi correo:
lx06156@innovacion.com

**Comment by Eduar2TC** [comment]
Yo también. Mi GitHub user es Eduar2TC. 
Agréguenme al repositorio del proyecto. No es necesario el email.

**Comment by GuidoSV7** [comment]
Yo igual busco, si gustan hacemos equipo 👀👀   ***@***.***

**Comment by Javilh97** [comment]
También estoy en búsqueda de equipo, les dejo mi correo si aun hay espacio, lx15413@innovaccion.mx

**Comment by WerlynRodriguez** [comment]
Hola buenas noches aún hay espacio 🥺? este es mi correo = lx14389@innovaccion.mx
Mi usuario en git es este mismo xd
